Index: afs-admin-tools/partinfo
===================================================================
--- afs-admin-tools.orig/partinfo	2024-07-23 18:11:46.524200015 +0000
+++ afs-admin-tools/partinfo	2024-07-23 18:14:13.979143165 +0000
@@ -39,12 +39,8 @@
 # Site configuration
 ##############################################################################
 
-# Test to find out whether the server is OpenAFS or YFS
+# If set assumes Auristor syntax
 our $YFS_SERVER;
-my $vos_version = `vos --version`;
-if ($vos_version =~ /^auristor/xms) {
-    $YFS_SERVER = 1;
-}
 
 # The default thresholds.  The first gives the threshold before the partition
 # is considered reasonably well-utilized, and defaults to 60%.  The second
@@ -175,6 +171,12 @@
     $server = 'afssvr' . $server;
 }
 
+# Check for AFS flavor
+my $vos_version = `vos --version`;
+if ($vos_version =~ /^auristor/xms) {
+    $YFS_SERVER = 1;
+}
+
 # Process threshold argument if provided.
 if ($thresholds) {
     if ($thresholds !~ /^(\d{1,2}|100),(\d{1,2}|100)$/) {
